src/init.cr

```diff
@@ -6,38 +6,13 @@ class LuckyCli::Init
   end
 
   def run
-    project_name = name
     if wants_to_generate_task_file?
       LuckyCli::Generators::TasksFile.run
-    elsif project_name
-      LuckyCli::Generators::Web.run(
-        project_name: project_name,
-        options: options
-      )
     else
       LuckyCli::Wizard::Web.run
     end
   end
 
-  private def options : Options
```

Comment:
We should still allow using these options but under a different command name. There are times using the CLI is helpful for stuff like this and a wizard would make things pretty hard. Maybe `lucky init.custom` or something. I don't really have a good name for it :S

Reply:
It's called `init.custom` for now. We can always rename later if we have a better name 😅